python/nav/web/templates/seeddb/management-profile/edit.html

```diff
@@ -41,20 +41,17 @@ <h4>Add new management profile</h4>
             </div>
         </div>
 
-        {% if object %}
-            {% with object.netbox_set.all.distinct.count as netbox_count %}
-                {% if netbox_count > 0 %}
-                    <div class="alert-box warning">
-                    <strong>{{ netbox_count }} IP device{{ netbox_count|pluralize:",s" }}</strong>
-                    will be affected by changes to this profile.
-                {% else %}
-                    <div class="alert-box info">
-                    No IP devices will currently be affected by changes to this profile.
-                {% endif %}
-                </div>
-            {% endwith %}
-        {% endif %}
-
+        {% if object %}{% if num_netboxes > 0 %}
+        <div class="alert-box warning">
+            <strong>{{ num_netboxes }} IP device{{ num_netboxes|pluralize:"s" }}</strong>
+            will be affected by changes to this profile.
+            (<a href="{% url "seeddb-netbox" %}?profile={{ object.pk }}">Link</a>)
```

Comment:
`Link` is *not* a good hyperlink text, but you already knew that.

My suggestion would be to hyperlink the entire "X IP devices" string instead.